src/content/docs/guides/setup-project.mdx

```diff
@@ -111,7 +111,7 @@ client.start();
     </TabItem>
 </Tabs>
 
-So after complete this steps your project structure should look like this:
+So after complete these steps, your project structure must look like this:
```

Comment:
aca lo dire en españo, en este caso "should" es correcto ya que lo que queremos decir es que "deberia de" y no "tiene que" ya que es posible que haya cometido algun error en el proceso

Reply:
Ya pero en este caso estas realizando todo lo anterior y al finalizar estas diciendo debería que puede referirse también a should pero must es como más fuerte y directo. Como en las anteriores ambas son posibles lo único que yo elegí lo que creía que podía quedar mejor. En este ejemplo estas enfatizando que como has hecho todo lo demás todo eso debe porque sino lo has seguido mal. Should es como un consejo o incluso algo opcional

Reply:
esto dejalo en should, me gusta mas porque es "deberia" y no " tienes que "